docs/_sass/custom/custom.scss

```diff
@@ -109,11 +109,12 @@ body {
 		font-weight: 600;
 		font-family: GalanoGrotesque-Bold;
 	}
-	h4,
-	.text-delta {
-		text-transform: none;
-		width: 100%;
-	    max-width: 264px;
+	.text-delta{
+		display: none;
+	}
```

Comment:
You hide the real table of content and inject your own using 'before'?
The CSS should content the style and not the content.

Reply:
Yes I hide it because we can't make table auto width based on content because the DOM structure for the table is not correct(table header and table content list are not in any container) so that's why I used before  for table list so it will auto width based on table content, hope that make sense

Reply:
The page layout is - https://github.com/treeverse/lakeFS/blob/master/docs/_layouts/default.html
You can change the structure of the toc if needed.